client/extensions/woocommerce/app/stats/index.js

```diff
@@ -2,19 +2,40 @@
  * External dependencies
  */
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
```

Comment:
Minor - we have been trying to keep the dependency import statements in alphabetical order elsewhere.